input/input.go

```diff
@@ -73,3 +74,33 @@ func SecureInput(input string) string {
 	}
 	return ""
 }
+
+// ParseMultilineInput ...
+func ParseMultilineInput(listInput string) []string {
+	trimmedInput := strings.TrimSpace(listInput)
+	if len(trimmedInput) == 0 {
+		return nil
+	}
+
+	separatedItems := []string{trimmedInput}
+	for _, separator := range []string{"\n", `\n`, "|"} {
```

Comment:
I think it would be better to use a single seperator here and remove "|" or make it optional.

Reply:
Ok I'll make "|" optional than to allow the original usage at https://github.com/bitrise-steplib/steps-google-play-deploy/blob/master/config.go#L95 to be kept as well.

Reply:
Is "\n", `\n`both needed?

Reply:
I'm not sure if both is needed, there is no info on this in google-play-deploy. Do you think we should keep both or should we keep only one?

Reply:
The idea may have been to support all line endings on all platforms (\r\n and \n), unsure if it is the right way to do this.

Reply:
I would say lets keep this as is. This has been working in google-play-deploy, at least there have been no issues reported on the appList input.